x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/fields_browser/fields_browser.spec.ts

```diff
@@ -117,9 +126,10 @@ describe('Fields Browser', () => {
     });
   });
 
-  context('Editing the timeline', () => {
+  context.only('Editing the timeline', () => {
```

Comment:
do we need `.only`?

Reply:
true! that was a mistake from my side, thx! 😊